src/pages/plans/Plans.jsx

```diff
@@ -1,73 +1,38 @@
 import * as React from 'react';
-import Paper from '@mui/material/Paper';
-import MenuList from '@mui/material/MenuList';
-import MenuItem from '@mui/material/MenuItem';
-import ListItemText from '@mui/material/ListItemText';
 import { Stack } from '@mui/material';
-import { Link } from 'react-router-dom';
-import { useParams } from 'react-router-dom';
-import { exercises } from '../../data/exercises';
-import Plan from './Plan';
+import Grid from '@mui/material/Grid';
+import ListItemButton from '@mui/material/ListItemButton';
+import { plansTabs } from '../../config';
 
 const Plans = () => {
-  const { category } = useParams();
+  const [activeTab, setActiveTab] = React.useState(plansTabs[0].id);
+
+  const ActiveTabComponent = plansTabs.find(
+    (item) => item.id === activeTab,
+  )?.content;
+
+  const handleClick = (tab) => {
+    setActiveTab(tab);
+  };
 
-  const relatedExercises = exercises.filter(
-    (item) => item.category.toLowerCase() === category,
-  );
-  console.log(relatedExercises);
   return (
-    <>
-      <Stack direction="row" my={8} spacing={2}>
-        <Paper sx={{ width: 320, maxWidth: '100%' }}>
-          <MenuList>
-            <MenuItem>
-              <ListItemText>
-                <Link
-                  style={{ textDecoration: 'none', color: 'green' }}
-                  to={'/plans/fullbody'}
-                >
-                  Full body
-                </Link>
-              </ListItemText>
-            </MenuItem>
-            <MenuItem>
-              <ListItemText>
-                <Link
-                  style={{ textDecoration: 'none', color: 'green' }}
-                  to={'/plans/hornicasttela'}
-                >
-                  Horní část těla
-                </Link>
-              </ListItemText>
-            </MenuItem>
-            <MenuItem>
-              <ListItemText>
-                <Link
-                  style={{ textDecoration: 'none', color: 'green' }}
-                  to={'/plans/dolnicasttela'}
-                >
-                  Dolní část těla
-                </Link>
-              </ListItemText>
-            </MenuItem>
-            <MenuItem>
-              <ListItemText>
-                <Link
-                  style={{ textDecoration: 'none', color: 'green' }}
-                  to={'/plans/kardio'}
-                >
-                  Kardio
-                </Link>
-              </ListItemText>
-            </MenuItem>
-          </MenuList>
-        </Paper>
-      </Stack>
-      {relatedExercises.map((exercise) => (
-        <Plan key={exercise.keys} exercise={exercise} />
-      ))}
-    </>
+    <Stack direction="row">
+      <Grid container>
+        <Grid item xs={4}>
+          {plansTabs.map(({ name, icon: Icon, id }) => {
+            return (
+              <ListItemButton onClick={() => handleClick(id)}>
```

Comment:
List items should be wrapped in some `List` component from MUI. There is also `ListItemIcon` component, that you can use for wrapping Icon.